Makefile.extra

```diff
@@ -0,0 +1,49 @@
+######################################################################-------------------------------------------------------------------------
```

Comment:
Please remove three files (Makefile.extra, run_base_tests.sh and run_extra_tests.sh)

You have added the following new files.
- Makefile.extra (exactly same as existing Makefile)
- run_base_tests.sh (almost same as existing test.sh except for sed REGRESS replacement)
- run_extra_tests.sh (almost same as existing test_extra.sh except for final line and exporting USE_PGXS)
However these are almost same with existing files, so I don't think these files are required.
If the problem is just that test.sh cannot be executed after text_extra.sh, I think we should modify test.sh adding sed REGRESS replacement.